### PR TITLE
feat: display createdAt in GE project details

### LINKED ISSUE #647: Add project created date to GE project details
**User Story:**
As a donor
I want to know how long a project has been around
So that I can assess its legitimacy

**Acceptance Criteria**
GIVEN that I am on a project's detail page on GE
WHEN I look in the project details section
THEN I see the date it was created

**Product & Design Links:**
[Figma](https://www.figma.com/file/lZwHrKYiWJna3AePzyTAiG/Grant-Explorer?node-id=1583%3A56059)

**Tech Details:**

**Open Questions:**

**Notes/Assumptions:**
- This includes writing test coverage for the functionality added in the ticket, if necessary

### PR DESCRIPTION
**Pull request by vacekj**
fixes #647

### CODE REVIEW

**Review by boudra — APPROVED** [review]

File: packages/grant-explorer/src/features/round/ViewProjectDetails.tsx [hunk 1]
```diff
@@ -186,6 +206,34 @@ function AboutProject(props: { projectToRender: Project }) {
   const userGithub = projectToRender.projectMetadata.userGithub;
   const projectGithub = projectToRender.projectMetadata.projectGithub;
 
+  const date = new Date(projectToRender.projectMetadata.createdAt ?? 0);
```
Comment (boudra) [inline packages/grant-explorer/src/features/round/ViewProjectDetails.tsx:209]: maybe we can split this into it's own utility function?

File: packages/grant-explorer/src/features/round/ViewProjectDetails.tsx [hunk 2]
```diff
@@ -33,6 +33,26 @@ import RoundEndedBanner from "../common/RoundEndedBanner";
 import PassportBanner from "../common/PassportBanner";
 import markdown from "../../app/markdown";
 
+const CalendarIcon = (props: React.SVGProps<SVGSVGElement>) => {
```
Comment (boudra) [inline packages/grant-explorer/src/features/round/ViewProjectDetails.tsx:36]: is this icon not available in heroicons? if we need it maybe we can split it into it's own file

Comment (hmrtn) [reply packages/grant-explorer/src/features/round/ViewProjectDetails.tsx:36]: Agreed - this is available in heroicons: https://github.com/tailwindlabs/heroicons/blob/master/src/24/solid/calendar.svg

**Review by hmrtn — CHANGES_REQUESTED** [review]

File: packages/grant-explorer/src/features/round/ViewProjectDetails.tsx (hunk 2 above)
Comment (vacekj) [reply packages/grant-explorer/src/features/round/ViewProjectDetails.tsx:36]: Unfortunately it's not the same icon. I tried this one originally, but it's not filled in like the one in Figma. Good idea on the split though.

**Review by melissa-neira — APPROVED** [review]

**Review by nfrgosselin — COMMENTED** [review]
Defer to @melissa-neira's approval

File: packages/grant-explorer/src/features/round/ViewProjectDetails.tsx (hunk 2 above)
Comment (vacekj) [reply packages/grant-explorer/src/features/round/ViewProjectDetails.tsx:37]: let's wait until we decided how we do icons in general for the splitting and ship this first

Comment (vacekj) [reply packages/grant-explorer/src/features/round/ViewProjectDetails.tsx:37]: @hmrtn

**Pull request merged**